site.conf

```diff
@@ -6,8 +6,15 @@
 	domain_seed = 'fcd2bb830c677865c60459899facdb630a9c09e26b72d995451c4feb9b337e6d',
 
 	opkg = {
+		openwrt = 'http://downloads.openwrt.org/releases/packages-%v/%A',
```

Comment:
is this just being explicit instead of implicit or does this change something?

Reply:
I *think* one has to fully fill the map with the default base (i.e upstream) definition when adding deriving package sources later on (i.e in `extra = [ … }`

Reply:
I haven't had the chance to test this change, yet.
according to readthedocs the default is snapshot not releases/packages-22.03

I didn't want to supply master packets.
run opkg update and upgrade once might actually update tons of packages for which we don't have the available flash.

it's possible that the documentation is wrong, I haven't checked the code

> openwrt overrides the default OpenWrt repository URL. The default URL would correspond to http://downloads.openwrt.org/snapshots/packages/%A and usually doesn’t need to be changed when nodes are expected to have IPv6 internet connectivity.

https://gluon.readthedocs.io/en/latest/user/site.html

@cbricart we could also use your mirror of openwrt here if it improves anything.
if it's just a forward then nvm
http://opkg.ffac.rocks/ (< read)

Reply:
actually I got the openwrt URL from Freifunk Darmstadt or Hanover 
https://github.com/freifunk-darmstadt/site-ffda/blob/master/site.conf#L23
https://github.com/freifunkh/site/blob/aba526211f7d1e49259c29dc701ba2a072e3db1d/site.conf#L9

Reply:
maybe it's actually possible to combine all of these feeds into one link

maybe I need a feeds.conf, like here? (maybe I don't and it just works)
https://downloads.openwrt.org/releases/packages-22.03/mips_24kc/

Reply:
Update:
```
/etc/opkg# cat distfeeds.conf 
# src/gz openwrt_core https://downloads.openwrt.org/releases/22.03-SNAPSHOT/targets/ath79/generic/packages
src/gz openwrt_base https://downloads.openwrt.org/releases/22.03-SNAPSHOT/packages/mips_24kc/base
src/gz openwrt_luci https://downloads.openwrt.org/releases/22.03-SNAPSHOT/packages/mips_24kc/luci
src/gz openwrt_packages https://downloads.openwrt.org/releases/22.03-SNAPSHOT/packages/mips_24kc/packages
src/gz openwrt_routing https://downloads.openwrt.org/releases/22.03-SNAPSHOT/packages/mips_24kc/routing
src/gz openwrt_telephony https://downloads.openwrt.org/releases/22.03-SNAPSHOT/packages/mips_24kc/telephony
```
This is how the OpenWrt feeds are setup, when we don't change anything.
If you take a look at:
https://downloads.openwrt.org/releases/22.03-SNAPSHOT, then you'll see, that packages is just a symlink
[packages](https://downloads.openwrt.org/releases/22.03-SNAPSHOT/packages/)/ -> ../packages-22.03

therefore changing the path above doesn't change a thing, actually
I'll therefore remove the line specifying openwrt again from our site.conf

furthermore:
I realized that the packages from the feeds base, packages and routing are just subsets from openwrt, therefore I'll remove those lines as well.
Which leaves: modules (our version of openwrt_core, which is commented out in distfeeds.conf for a reason)
and ffac, gluon and gluon_base

Those will all be defined/listed in /etc/opkg/gluon.conf (right now it contains only modules)

is mistook snapshots for master-based packages btw.
while it actually meant snapshots of the 22.03 branch, which is fine